service/apiserver/internal/cache/mem_cache.go

```diff
@@ -260,9 +261,30 @@ func (m *MemCache) GetAssetNameById(assetId int64) (string, error) {
 	}
 
 	m.goCache.Set(key, asset.AssetName, gocache.DefaultExpiration)
+	key2 := fmt.Sprintf("%s%d", AssetIdSymbolKeyPrefix, assetId)
+	m.goCache.Set(key2, asset.AssetSymbol, gocache.DefaultExpiration)
+
 	return asset.AssetName, nil
 }
 
+func (m *MemCache) GetAssetSymbolById(assetId int64) (string, error) {
+	key := fmt.Sprintf("%s%d", AssetIdSymbolKeyPrefix, assetId)
+	name, found := m.goCache.Get(key)
+	if found {
+		return name.(string), nil
+	}
+	asset, err := m.assetModel.GetAssetById(assetId)
+	if err != nil {
+		return "", err
+	}
+
+	m.goCache.Set(key, asset.AssetSymbol, gocache.DefaultExpiration)
+	key2 := fmt.Sprintf("%s%d", AssetIdNameKeyPrefix, assetId)
```

Comment:
can we have better name for the `key` and `key2`

Reply:
ok, let me fix